lib/emergencyNotification.js

```diff
@@ -0,0 +1,222 @@
+var Client = require("./client");
+var EMERGENCY_NOTIFICATION_RECIPIENTS = "emergencyNotificationRecipients";
+var EMERGENCY_NOTIFICATION_GROUP_ORDERS = "emergencyNotificationGroupOrders";
+var EMERGENCY_NOTIFICATION_GROUPS = "emergencyNotificationGroups";
+var EMERGENCY_NOTIFICATION_ENDPOINT_ORDERS = "emergencyNotificationEndpointOrders";
+
+function EmergencyNotification(){
```

Comment:
Again probably could be an object, rather than a function.

Reply:
This was done to keep in the practice of how the SDK was written so there no varying styles.  But if the SDK is every refactored a class would be a better way.